CLAUDE.md

```
@@ -147,4 +147,5 @@ When creating new UI components:
 1. Use `CCHaksaTheme` for new features
 2. Preview components in `preview/designsystem/` package
 3. Follow existing color naming conventions (100-600 scale)
-4. Prefer typography styles over hardcoded text styles
\ No newline at end of file
+4. Prefer typography styles over hardcoded text styles
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)